src/sage/functions/other.py

```diff
@@ -501,7 +501,7 @@ def __init__(self):
             sage: a = floor(5.4 + x); a
             floor(x + 5.40000000000000)
             sage: a.simplify()
-            floor(x + 0.4000000000000004) + 5
+            floor(x + 0.400000000000000...) + 5
```

Comment:
This looks like it works without the fuzz (because 1/4 is a power of two):

```
sage: a = floor(5.25 + x); a
floor(x + 5.25000000000000)
sage: a.simplify()
floor(x + 0.25) + 5
```

but the best example is probably,

```
sage: a = floor(27/5 + x); a
floor(x + 27/5)
sage: a.simplify()
floor(x + 2/5) + 5
```